plat/Config.uk

```diff
@@ -115,3 +115,7 @@ config FPSIMD
 	depends on ARCH_ARM_64
 	help
 		Enable support FPU usage in application
+
+config UKPLAT_AUXSP_SIZE
```

Comment:
In other cases we use order instead of size in bytes:
```
CONFIG_STACK_SIZE_PAGE_ORDER=8
CONFIG_LIBPOSIX_PIPE_SIZE_ORDER=16
CONFIG_LIBUKFALLOCBUDDY_MAX_ALLOC_ORDER=30
CONFIG_LIBVFSCORE_PIPE_SIZE_ORDER=16
```

Reply:
Bumping for the next update.